datadog-monitors-monitor-handler/src/datadog_monitors_monitor/version.py

```diff
@@ -0,0 +1 @@
+__version__ = "2.2.0"
```

Comment:
Can we do a beta.1 ?

Reply:
Actually this should even be
```suggestion
__version__ = "2.2.0.dev"
```